main/src/test/java/cgeo/geocaching/utils/formulas/FormulaTest.java

```diff
@@ -60,6 +60,15 @@ public void bigIntegers() {
         assertThat(Formula.evaluate("cs(" + longMaxBeyond + ")").getAsInt()).isEqualTo(longMaxCs + 1);
     }
 
+    @Test
+    public void bigDecimals() {
+        final int doublePowCs = 847; // manually calculated checksum of pow(99^99)
+        assertThat(Formula.evaluate("cs(99^99)").getAsDouble()).isEqualTo(doublePowCs);
+        assertThat(Formula.evaluate("99^99").getAsDouble()).isEqualTo(Math.pow(99, 99));
+        assertThat(String.valueOf(Formula.evaluate("99^99").getAsString())).isEqualTo("369729637649726802192985226395427290145296428445515959701359650120802601667133273280053721002700400354392780458116125965728631706472588849812738072765460822138161108630185181415759762204338929270784");
+        assertThat(String.valueOf(Formula.evaluate("99^99").getAsDouble())).isEqualTo("3.697296376497268E197");
```

Comment:
Assert should be for double value, not for its string representation

Reply:
Thats in line 67...

Reply:
Even not sure, if it is the same on all machines..🤔